compiler/tflchef/core/src/OpUtils.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "OpUtils.h"
+
+#include <stdexcept>
+
+void check_custom_op_value(const tflchef::Operation operation, std::string op_type)
+{
+  if ((operation.has_extype() && operation.extype() == "Custom") ||
+      (operation.has_type() && operation.type() == "Custom"))
```

Comment:
As I know, `ex_type` is optional and `operation.type()` can be null, and we will remove this later.
But do we have to call `operation.has_type()` too?

Reply:
> But do we have to call `operation.has_type()` too?

No. Come to think of it, it doesn't seem necessary. :)
I pushed the code reflected in your comment.
Could you review the code again??